Tella/Scenes/Settings/ViewModel/SettingsViewModel.swift

```diff
@@ -69,6 +69,12 @@ class SettingsViewModel: ObservableObject {
     }
     
     func saveDeleteAfterFail() {
+        if(appModel.settings.deleteAfterFail == .off && selectedDeleteAfterFailOption != .off) {
```

Comment:
or simply you can write 

        if selectedDeleteAfterFailOption != .off {
            appModel.settings.showUnlockAttempts = true
        }
       
 because in the case of  `selectedDeleteAfterFailOption` equal to `off` we don't check `showUnlockAttempts`
        
or 
       
 simply  `appModel.settings.showUnlockAttempts = selectedDeleteAfterFailOption != .off ? true : false`

Reply:
Yeah I thought of this at the beginning but there is this edge case:

- User has delete after failed set to 5 attempts
- User switch toggle to off
- User change from 5 attempts to 10 attempts

In this case I think the toggle just need to remain off. And with this logic: `appModel.settings.showUnlockAttempts = selectedDeleteAfterFailOption != .off ? true : false` It will be toggle to true everytime the user change the number of attempts. 


I will double-check with Caro and Yann Chwenn to know what the correct behavior is.